pkg/sql/sem/tree/format.go

```diff
@@ -83,8 +83,8 @@ const (
 	FmtSymbolicSubqueries
 
 	// If set, strings will be formatted for being contents of ARRAYs.
-	// Used internally in combination with FmtArrays defined below.
-	fmtWithinArray
+	// Used internally in combination with FmtPgwireText defined below.
+	pgwireFormat
```

Comment:
I'm not especially convinced that this shouldn't be its own method or function (on `Datum`) at this point, rather than a flag on `Format`, but I don't feel particularly strongly.